pyrobosim/pyrobosim/utils/knowledge.py

```diff
@@ -8,7 +8,9 @@
 import numpy as np
 
 
-def apply_resolution_strategy(world, entity_list, resolution_strategy, robot=None):
+def apply_resolution_strategy(
+    world, entity_list: list, resolution_strategy: str, robot=None
+):
```

Comment:
Can you undo these type hints? I'd rather do them all in one shot at some point, and having 2/4 args with them feels kind of off.